libevmasm/Instruction.h

```diff
@@ -103,6 +103,9 @@ enum class Instruction: uint8_t
 	GAS,				///< get the amount of available gas
 	JUMPDEST,			///< set a potential jump destination
 
+	TLOAD = 0x5c,       ///< https://eips.ethereum.org/EIPS/eip-1153
+	TSTORE = 0x5d,      ///< https://eips.ethereum.org/EIPS/eip-1153
```

Comment:
I think it's more useful to have actual descriptions here.
```suggestion
	TLOAD = 0x5c,       ///< load word from transient storage (EIP-1153)
	TSTORE = 0x5d,      ///< save word to transient storage (EIP-1153)
```

 If you really want, you could include the EIP in parentheses.

Reply:
I will remove it altogether.